packages/utils/src/utils/split-utxos.test.ts

```diff
@@ -0,0 +1,52 @@
+import { FuelError, ErrorCode } from '@fuel-ts/errors';
+import { expectToThrowFuelError } from '@fuel-ts/errors/test-utils';
+import type { AbstractAddress } from '@fuel-ts/interfaces';
+import { BN } from '@fuel-ts/math';
+
+import { splitUTXOs } from './split-utxos';
+
+/**
+ * @group node
+ * @group browser
+ */
+describe('splitUTXOs', () => {
+  it('should generate two UTXOs by default', () => {
+    // using unknown as AbstractAddress to avoid importing @fuel-ts/address as that's a circular dependency
+    const destination = '0x...' as unknown as AbstractAddress;
```

Comment:
This indicates that we should define this helper somewhere else. We can put it within the `account` package. The result will be the same since users will import it from `fuels`.

Reply:
I disagree, this is a utility specific to UTXOs and had I chosen to accept the `destination` param as a string this wouldn't be necessary. 
I could move it to the account to avoid this cast but I don't think structurally this is a function of the account.

Reply:
No longer relevant 7165a1a85